e2e/src/app.e2e-spec.ts

```diff
@@ -55,6 +82,7 @@ describe('Band detail page', () => {
   });
 
   it('should display band training content when a category is clicked', () => {
+  browser.driver.sleep(500);
```

Comment:
us there not a better way?

Reply:
Couldn't find a better method for fixing this within the time limits